### PR TITLE
Add refresh nfts button

### LINKED ISSUE #106: [Task]: Add refresh button to NFTs section
**Task description.**

Add refresh button to NFTs section
image

**Requirements.**
- Add button
- Refresh minted NFTs
- Update store

### PR DESCRIPTION
**Pull request by cpl121**
Fixes #106

## Changes

  - Refresh minted NFTs
  - Update store
  
### Checks

* [X] Have you catchup your branch with the latest state in base?
* [X] Have you lint your code locally prior to submission?
* [X] Have you reviewed your proposed changes and removed debris?
* [X] Have you successfully ran due tests with your changes?

### CODE REVIEW

File: pages/[id].tsx [hunk 1]
```diff
@@ -23,6 +34,8 @@ const CandyMachine: NextPage = () => {
     const [error, setError] = useState('')
     const [nfts, setNfts] = useState<Nft[]>([])
     const [isLoading, setIsLoading] = useState(false)
+    const [isLoadingRefresh, setIsLoadingRefresh] = useState(false)
```
Comment (VmMad) [inline pages/[id].tsx:37]: I think it's better to call this isRefreshing or isReloading

File: pages/[id].tsx [hunk 2]
```diff
@@ -78,6 +91,22 @@ const CandyMachine: NextPage = () => {
         }
     }
 
+    const refreshNfts = async () => {
+        setIsLoadingRefresh(true)
+        try {
+            if (candyMachineAccount) {
+                const nfts = await getAllNftsByCM(candyMachineAccount, connection)
+                console.log(nfts)
```
Comment (VmMad) [inline pages/[id].tsx:99]:
```suggestion
```

Comment (cpl121) [reply pages/[id].tsx:99]: 🤕

**Review by VmMad — APPROVED** [review]
LGTM!

**Review by evavirseda — APPROVED** [review]
Works fine!

**Pull request merged**